interfaces/HO-Portal/src/app/program/program-payout/program-payout.component.html

```diff
@@ -35,6 +38,16 @@ <h3>{{ 'page.program.program-payout.make-export'|translate }}</h3>
       [paymentInstallment]="exportInstallmentId"
       [disabled]="!exportInstallmentAvailable"
     ></app-export-list>
+    <ng-container>
```

Comment:
The `ng-container` isn't really required here, I think, if there is no real condition-attribute on it.

(I'm not sure why it is used inside the `<option>`-elements above... :? )